addons/event/models/event_mail.py

```diff
@@ -80,6 +80,12 @@ class EventMailScheduler(models.Model):
     def _selection_template_model(self):
         return [('mail.template', 'Mail')]
 
+    @api.onchange('notification_type')
+    def set_template_ref_model(self):
+        mail_model = self.env['mail.template']
+        if self.notification_type == 'mail':
+            self.template_ref = "{},{}".format('mail.template', mail_model.search([], limit=1).id)
```

Comment:
couldn't we set the value to false instead?

this seems ok too, but it seems it was the default behavior when the code worked (when you were administrator)